README.md

```diff
@@ -127,6 +132,31 @@ with the following additions:
 
 > **NOTE**: Feel free to use another workspace folder name.
 
+### Without dev container
+
+#### Install Build Dependencies
+
+As stated above, the `devcontainer.json` and the Dockerfile
+`.devcontainer/Dockerfile` contain a list of the plugins/tools we use for Chariott.
+Below we have listed the steps to get started, but refer to those files if there are any discrepancies.
+
+- Install [rust](https://rustup.rs/#)
+- Install [cmake](https://cmake.org/install/)
```

Comment:
Is cmake needed for building chariott outside of a dev container?

Reply:
Rust needs a C linker. Our dockerfile currently uses cmake, so I will leave this as-is for now, but we can investigate showing the more broad "C linker" dependency or listing gcc instead.